doc/lstm.txt

```diff
@@ -174,7 +174,7 @@ be computed with :
 
 .. math::
 
-    z = \sigma(W x_t + U h_{t-1} + b)
+    z = W x_t + U h_{t-1} + b
```

Comment:
Can you double check that the code do that? I think so, but I could have looked at the wrong place in the code.